ammonia-CO-H2-Alzueta-2023.yaml

```diff
@@ -0,0 +1,2549 @@
+description: |-
+  A variation of the 2023 model of NH3 and H2 chemistry published by Alzueta and coworkers, with the original version denoted below as "baseline". [1] In the modified version, "linear-Burke", six complex-forming reactions have been modified according to the linear Burke rate implementation of the reduced-pressure linear mixture rule (LMR-R). These are: H2O (+M), HO2 (+M), H2O2 (+M), NH3 (+M), N2H4 (+M), and HNO (+M). Ab initio third-body efficiencies were obtained at discrete temperatures from Jasper [2] and then fitted to an Arrhenius parameterization to facilitate the temperature dependence of these parameters. Further description of the methodology used to adapt this mechanism is available in Singal et al. [3]
```

Comment:
As we do everywhere, please line wrap to 88 characters.

Reply:
I have set it to linewrap to 88 characters in VScode, but perhaps this doesn't actually change anything because it's just a user setting. What is your strategy for line wrapping?

Reply:
Yes, I mean the number of characters in an actual line, not what you've set your editor to do to very long lines.

For VS Code, I would recommend setting a ruler at the 88th column (using the [`editor.rulers`](https://stackoverflow.com/questions/29968499/how-can-i-have-multiple-vertical-rulers-in-vs-code) setting).

And to help automatically wrap long text blocks to fit in that space, I use the [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) extension.